### PR TITLE
Bump version to 0.24.1

### PR DESCRIPTION
**Pull request by ahukkanen**
In order to release a new patch version of the bulletin board packages/gems and the voting schemes, I am bumping the version number to 0.24.1 and adding a CHANGELOG entry for this.

The main reason for the new release is the OpenSSL 3.0 fix implemented at #294.

We need this release in order to finish decidim/decidim#9766.

### CODE REVIEW

**Comment by ahukkanen** [comment]
@andreslucena I need one approving review to be able to merge this, so I'd appreciate if you could check it.

I am only bumping the version to 0.24.1 (through `VERSION="0.24.1" make bump_versions`) in order to release the new version with the important fix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.24.1] - 2022-11-11
+
+## Fixed
+
+- Fixed issue with bulletin board Ruby client OpenSSL 3.0 compatibility
```
Comment (andreslucena) [inline CHANGELOG.md:14]: This was manually added? Should we add the other changes too? 

https://github.com/decidim/decidim-bulletin-board/compare/v0.24.0...develop

**Review by andreslucena — CHANGES_REQUESTED** [review]
I have only one doubt regarding the CHANGELOG and it'd be ready to go for me.

File: CHANGELOG.md (hunk 1 above)
Comment (ahukkanen) [reply CHANGELOG.md:14]: Yes, this is manually created as we don't have automation in this repo.

Aren't they mostly documentation / local configuration changes?

I think they are not relevant in the changelog, I would categorize them in the Decidim main repo as `internal`.

Also looking at the previous entries in the changelog, I don't think it includes everything.

Comment (andreslucena) [reply CHANGELOG.md:14]: Ok, maybe we'd need to standardize the labels/changelog generation/etc to be consistent with `decidim/decidim`. But at the moment as this project isn't used in production I think that what we have is good then.

**Review by andreslucena — APPROVED** [review]
👍🏽

**Pull request merged**